R/network.R

```diff
@@ -77,36 +69,32 @@ transform_gitlog_to_hdsmj <- function(project_git, hdsmj_path, is_sorted=FALSE){
   graph_to_dsmj(cochange_table, hdsmj_path, dsmj_name="hdsm", is_directed=FALSE, is_sorted)
 }
 
-#' Transform parsed git repo into a adsmj file.
+#' Transform parsed git repo into an author dsm.json file.
 #'
 #' Converts a temporal gitlog table into an *-adsm.json.
-#' In the hdsm.json, the Variables are all files/methods or any variables under analysis
+#' In the adsm.json, the Variables are all the authors under analysis
 #' (rows/columns in dependency matrix) and the Cells (matrix cell) contain all the relations of
-#'  variable (src & dest) pairs. The Co-change is the number of times the src & dest
-#'   were committed together.
-#' Note that the cochange between a file and its renamed variant will not be considered
-#' using this function, so those cells won't appear in the final *-hdsm.json.
+#'  variable (src & dest) pairs. The Collaborate value is the number of times the src author and dest author changed the same file.
```

Comment:
I'm not sure if my definition of the Collaborate value is the best.

Reply:
I think this is fine!